articles/scp-3211/src/config.ts

```diff
@@ -1,5 +1,6 @@
 export const anomalyNames = <const>[
   "base", // Base anomaly must come first
+  "cat",
```

Comment:
Anomalies should be declared in the order they're added - this list determines what order the buttons appear in on the selection menu, and I want older ones to appear first

Reply:
Ah I see, apologies. Maybe leave a comment saying "add at the end"?